Alexandroff square isn't homogeneous. (#943)
pi-base/data: spaces/S000099/properties/P000086.md

---
space: S000099
property: P000086
value: false
---

> Points of the diagonal don't have a countable base, but points out of the diagonal do. (See {S99|P28}.)

Reviewer: ```suggestion
Points on the diagonal don't have a countable local base, but points off the diagonal do. (See {S99|P28}.)
```
